source/android/adaptivecards/build.gradle

```diff
@@ -124,11 +120,14 @@ afterEvaluate {
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
+    implementation 'androidx.gridlayout:gridlayout:1.0.0'
     androidTestImplementation('androidx.test.espresso:espresso-core:3.1.0', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
     implementation 'androidx.appcompat:appcompat:1.0.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'com.google.android:flexbox:1.0.0'
     testImplementation 'junit:junit:4.12'
+    implementation "androidx.core:core-ktx:1.5.0-rc02"
```

Comment:
No, not required. As our partners use Kotlin, we can start to leverage features to make a better API (e.g. marking some fields as `internal` instead of making everything `public`, marking required parameters for null safety, etc.). So, for the two new classes here, I've used Kotlin to kickstart that migration.

Reply:
in the case of APIs, are java and kotlin interchangeable? would there be any breaking changes if we fully migrate to kotlin?